pretty_midi/pretty_midi.py

```diff
@@ -456,7 +519,7 @@ def get_onsets(self):
         # Return them sorted (because why not?)
         return np.sort(onsets)
 
-    def get_piano_roll(self, fs=100, times=None):
+    def get_piano_roll(self, fs=100, times=None, include_pitch_bends=True):
```

Comment:
Looks like you didn't add `include_pitch_bends` to the docstrings.  You'll need to rebase first before you do that anyways.  Or, we can handle that in a separate pull request, as it's independent of the key/time signature stuff.